### PR TITLE
[minor] Support IBM Cloud COS buckets and Manage logging feature

### PR DESCRIPTION
**Pull request by Ashish-Mahawal1**
Since the work around COS was comprehending multiple stories, I got this PR to cover roles migration and integration for the following:

- sre.devops.cos_bucket into mas_devops.cosbucket [#10617](https://github.ibm.com/wiotp/tracker/issues/10617)
- sre.ibmcloud_cos into mas_devops.cos [#10872](https://github.ibm.com/wiotp/tracker/issues/10872)
- sre.manage_logging_setup into mas_devops.suite_manage_logging_config [#10580](https://github.ibm.com/wiotp/tracker/issues/10580)

This PR only covers the support to IBM Cloud, S3/AWS support will be covered in part 2 (next sprint)

### CODE REVIEW

**Review by andrercm — COMMENTED** [review]
Looks good! I just didn't found the assertions piece to validate whether the cos instance was provisioned or deprovisioned at the end of the process.

**Comment by Ashish-Mahawal1** [comment]
> Looks good! I just didn't found the assertions piece to validate whether the cos instance was provisioned or deprovisioned at the end of the process.

Added those assertions as well :)

**Review by durera — CHANGES_REQUESTED** [review]
Also needs updates to the changelogs as this is a minor update

File: ibm/mas_devops/roles/cos_bucket/README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+cos_bucket
+===
+This role
```
[the remark this answers is not in the document]
Comment (andrercm) [reply ibm/mas_devops/roles/cos_bucket/README.md:3]: Sorry, didn't notice that before final commit, that's what i get when working on docs while joining lots of calls lol
Just fixed it

File: ibm/mas_devops/roles/suite_app_config/tasks/main.yml [hunk 2]
```diff
@@ -57,6 +56,10 @@
       - "MAS app namespace ......... {{ mas_app_namespace }}"
       - "Templated workspace CR .... {{ lookup('template', 'templates/workspace.yml.j2') }}"
 
+- name: Gen certs file
```
[the remark this answers is not in the document]
Comment (andrercm) [reply ibm/mas_devops/roles/suite_app_config/tasks/main.yml:59]: Initially added this to debug the output of the manageworkspace cr, wont use at all though, so just cleaned it

[the remark this answers is not in the document]
Comment (andrercm) [reply ibm/mas_devops/roles/suite_app_config/tasks/main.yml]: Done

**Review by durera — APPROVED** [review]

**Pull request merged**